app.js

```diff
@@ -0,0 +1,51 @@
+$(document).ready((e) => {
+
+    const key = '643a630e1616ba355e88b51064857533';
+    // Key should reside on server.
```

Comment:
Yes, it would be important to keep this private. For now, I would suggest starting with the following:

1. Make a file called `apiKeysTemplate.js` which looks something like:

```
myL3 = myL3 || {};
myL3.apiKeys = {
    openweathermapKey = "REPLACE THIS";
};
```

2. Add instructions to the `README.md` saying 'To configure the API keys, copy `apiKeysTemplate.js` to `apiKeys.js` and replace the API key placeholders "REPLACE THIS" with the appropriate keys.'
3. Add `apiKeys.js` to `.gitignore` so that we don't accidentally commit it
4. Read in the `apiKeys.js` file in `index.html`.

Reply:
Oh Thanks! This is very interesting.